Essentials/src/com/earth2me/essentials/commands/Commandwhois.java

```diff
@@ -48,5 +63,44 @@ public void run(final Server server, final CommandSource sender, final String co
         sender.sendMessage(tl("whoisJail", (user.isJailed() ? user.getJailTimeout() > 0 ? DateUtil.formatDateDiff(user.getJailTimeout()) : tl("true") : tl("false"))));
         sender.sendMessage(tl("whoisMuted", (user.isMuted() ? user.getMuteTimeout() > 0 ? DateUtil.formatDateDiff(user.getMuteTimeout()) : tl("true") : tl("false"))));
 
+        InetSocketAddress addr = getProxyAddress(Bukkit.getServer().getPlayer(user.getConfigUUID()));
+        if( isBungeeCord() && addr != null ) {
```

Comment:
Why not check if the server is bungee before getting the proxy address? At this point, the check is useless because if it isn't bungee, addr will be null.